modules/getting-started/pages/sync-api-apisync.adoc

```diff
@@ -1,21 +0,0 @@
-= To Sync an API
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-:noindex:
-
-When you update a RAML file from Anypoint Studio, you can synchronize those changes to your API version hosted in Anypoint Platform. Assuming you created the placeholder API described in the previous sections, make some small change to the API. Navigate to your `placeholder.raml` file in the Package Explorer, and double-click to open the file in your RAML Editor. For example, edit the RAML to change a description slightly.
```

Comment:
"to your API version" > "with your API version"
Or maybe "execute" or "import" or some other term instead of "synchronize to"?

Reply:
great comments! I'm not taking action because we're deleting this file.